Libplanet/Net/Swarm.BlockSync.cs

```diff
@@ -172,14 +172,9 @@ CancellationToken cancellationToken
                 var canonComparer = BlockChain.Policy.CanonicalChainComparer;
                 if (synced is { } syncedB
                     && !syncedB.Id.Equals(blockChain?.Id)
-                    && (!(blockChain.Tip is { } tip && syncedB.Tip is { } syncedTip)
-                        || canonComparer.Compare(
-                            blockChain.PerceiveBlock(tip),
-                            blockChain.PerceiveBlock(
-                                syncedTip,
-                                syncedB.PerceiveBlock(syncedTip).PerceivedTime
-                            )
-                        ) < 0
+                    && (canonComparer.Compare(
+                        blockChain.PerceiveBlock(blockChain.Tip),
+                        blockChain.PerceiveBlock(syncedB.Tip)) < 0
```

Comment:
Is it okay without `PerceivedTime`?

Reply:
I think it's okay because [`BlockChain<T>.PerceiveBlock()` uses `perceivedTime` only when the given perception is not found on its own storage.](https://github.com/planetarium/libplanet/blob/2eaab5a8b12a6ead49f8f822abe00396d18a55ba/Libplanet/Blockchain/BlockChain.cs#L732-L738) cc @dahlia

Reply:
🆗